src/components/game.js

```diff
@@ -1,72 +1,94 @@
-import React from 'react';
+import React, { Component } from "react";
 
-import Header from './header';
-import GuessSection from './guess-section';
-import GuessCount  from './guess-count';
-import GuessList from './guess-list';
+import Header from "./header";
+import GuessSection from "./guess-section";
+import StatusSection from "./status-section";
+import InfoSection from "./info-section"
 
-export default class Game extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            guesses: [],
-            feedback: 'Make your guess!',
-            correctAnswer: Math.floor(Math.random() * 100) + 1,
-        };
+export default class Game extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      guesses: [],
+      feedback: "Make your guess!",
+      auralStatus: '',
+      correctAnswer: Math.round(Math.random() * 100) + 1,
+      gameWon: false
+    };
+  }
+
+  restartGame() {
+    this.setState({
+      guesses: [],
+      feedback: "Make your guess!",
+      auralStatus: '',
+      correctAnswer: Math.floor(Math.random() * 100) + 1,
+      gameWon: false
```

Comment:
This can be derived from our existing state, so doesn't need to have its own variable:

```js
const gameWon = this.state.guesses[this.guesses.length - 1] === this.state.correctAnswer
```